tests/network/localnet/conftest.py

```diff
@@ -243,13 +292,18 @@ def vm_ovs_bridge_localnet_1(
     cudn_localnet_ovs_bridge: libcudn.ClusterUserDefinedNetwork,
     unprivileged_client: DynamicClient,
 ) -> Generator[BaseVirtualMachine]:
+    networks = [Network(name=LOCALNET_OVS_BRIDGE_NETWORK, multus=Multus(networkName=cudn_localnet_ovs_bridge.name))]
+    interfaces = [Interface(name=LOCALNET_OVS_BRIDGE_NETWORK, bridge={})]
+    network_data = cloudinit.NetworkData(
+        ethernets={PRIMARY_INTERFACE_NAME: cloudinit.EthernetDevice(addresses=[next(ipv4_localnet_address_pool)])}
+    )
```

Comment:
This looks similar to the previous one [1], please follow the same pattern.|

Ref: https://github.com/RedHatQE/openshift-virtualization-tests/pull/2370#discussion_r2507756854